lib/ignored-paths.js

```diff
@@ -92,21 +69,81 @@ function IgnoredPaths(patterns) {
  * @returns {IgnoredPaths} object, with patterns loaded from the ignore file
  */
 IgnoredPaths.load = function(options) {
-    var patterns;
+    var ignoredPaths = new IgnoredPaths();
+    ignoredPaths.loadFile(options);
+    return ignoredPaths;
+};
 
-    options = options || {};
+/**
+ * Reset IgnoredPaths
+ * @returns {void}
+ */
+IgnoredPaths.prototype.reset = function() {
```

Comment:
Will remove. It was used in a unit test once, but instead of a reset creating a new instance is the better way.